### PR TITLE
Create projectlist.html

### PR DESCRIPTION
**Pull request by Arvindjitta**
This is a html page for showing all community repositories at one place in the card format

note: as of now the above html works only in lg (large screens)

pr

### CODE REVIEW

**Comment by eddiejaoude** [comment]
I would probably add it to the `index.html` for now

**Comment by eddiejaoude** [comment]
Can you add a screenshot to the PR please

**Comment by Arvindjitta** [comment]
> I would probably add it to the `index.html` for now

as we discussed earlier that their are some things to be changed in index.html ...so i made separate html page for cards

**Comment by eddiejaoude** [comment]
> as we discussed earlier that their are some things to be changed in index.html 

what is to be changed? It doesn't add value in another file with no link to it 😞

**Comment by Arvindjitta** [comment]
> > as we discussed earlier that their are some things to be changed in index.html
> 
> what is to be changed? It doesn't add value in another file with no link to it 😞

some svg links has to be edited

**Comment by eddiejaoude** [comment]
But that is not related to your PR. Your PR will go below it 😉

**Comment by eddiejaoude** [comment]
I would add the basic cards in the `index.html` file, that has useful information listing all the community organisations projects, with the following information:

- project name
- project description
- repo link
- website link
- tech stack
- status

*note: dont worry if you are not sure of the `status` or `tech stack`, just put in a place holder and I can update these*

File: projectlist.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (alrifay) [inline projectlist.html:8]: I noticed you are using material Icons and in `index.html` they are using font awesome 
https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/blob/c9533c33c8c5f99b9a580bd3d7dc73f71faeb1ac/index.html#L11-L14

@eddiejaoude Is it fine to use multiple icon libraries ?

Comment (eddiejaoude) [reply projectlist.html:8]: Great spot @alrifay 👍  . You are right, we do not need another icon library included. Font-awesome has enough icons

**Comment by Arvindjitta** [comment]
> I would add the basic cards in the `index.html` file, that has useful information listing all the community organisations projects, with the following information:
> 
> * project name
> * project description
> * repo link
> * website link
> * tech stack
> * status
> 
> _note: dont worry if you are not sure of the `status` or `tech stack`, just put in a place holder and I can update these_

shall i make new pull request by making changes to index.html??

**Comment by eddiejaoude** [comment]
> shall i make new pull request by making changes to index.html??

Up to you, you can push more commits to this PR or create a new PR - I dont mind, whichever is easiest for you 🤓

**Comment by Arvindjitta** [comment]
I think It's better to maintain one page i'll delete this PR and make changes to index.html

**Comment by eddiejaoude** [comment]
Sure, you can just `close` this PR and create a new one 🤓

**Comment by eddiejaoude** [comment]
Dont worry. I can do it. 

New PR created https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/pull/21

**Pull request closed without merging**